### DISCUSSION TITLE
NPM的0.3.14是不是缺了scripts文件夹？

### DISCUSSION THREAD

**Discussion by FidStyle** [theme-shoka-x/hexo-theme-shokaX#227] [General]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/d011a948-469c-41d4-b3a5-7bc3bce8ee4d)
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/d0d882e5-4f7b-4e9c-b793-6a247066412f)

**Comment by FidStyle** [comment]
@zkz098

**Reply by zkz098** [reply]
手动发布脚本有问题，没编译ts，马上修

**Comment by FidStyle** [comment]
> 手动发布脚本有问题，没编译ts，马上修
